app/jobs/check_catalogues_job.rb

```diff
@@ -43,15 +40,15 @@ def catalogue_gather
         set_catalogue_region(region)

```

Comment:
Comments on line 29 and 36 can be safely removed, the line below them explains what's happening. 

Consider having line 37 use a method that generates the shop catalogues url.